### PR TITLE
build: bump discord-api-types version

### PR DESCRIPTION
**Pull request by Yareaj**
**Please describe the changes this PR makes and why it should be merged:**
Bump `discord-api-types` to `0.37.103`

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

File: package-lock.json
Comment (almeidx) [inline package-lock.json:1]: We don't use npm. This should be removed

Comment (Yareaj) [reply package-lock.json:1]: I didn't intend to commit that mb 💀

**Review by almeidx — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by didinele — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Comment by Jiralite** [comment]
Update your branch as it's out of date @Yareaj.

**Comment by Yareaj** [comment]
@Jiralite done!

**Pull request merged**